RachelAbaniwo/output-testing-and-debugging
# output(testing-and-debugging output)

#### What does this PR do?
Test a function that takes a number or an array of numbers as its parameter, if the parameter is an array of numbers, the function returns an object containing two arrays, one array containing the even numbers and another the odd numbers in the array, but if the parameter is a number, the function returns an object containing two arrays, one array containing the even numbers and another the odd numbers, these odd and even numbers are gotten from numbers between zero and the input parameter.

#### Description of Task to be completed?
- install jest
- install babel-jest and regenerator-runtime
- create .babelrc file at the root folder
- add test script to package.json file
- write tests
- create readme.md file

#### How should this be manually tested?
* Clone the Repository:
```typescript
$ git clone https://github.com/RachelAbaniwo/output-testing-and-debugging.git
```
* Make More-Recipes your current directory:
```terminal
$ cd /output-testing-and-debugging
```
* install all necessary dependencies with:
```typescript
$ npm install
```
* run `npm run test`
* tests should fail initially
* write function and link the function to the test file
* the test can then be used to test the function file

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
